integreat_compass/core/settings.py

```diff
@@ -309,3 +309,59 @@
         },
     },
 }
+
+##########
+# EMAILS #
+##########
+
+#: The backend to use for sending emails (see :setting:`django:EMAIL_BACKEND` and :doc:`django:topics/email`)
+EMAIL_BACKEND = (
+    "django.core.mail.backends.console.EmailBackend"
+    if DEBUG
+    else "django.core.mail.backends.smtp.EmailBackend"
+)
+
+#: Default email address to use for various automated correspondence from the site manager(s)
+#: (see :setting:`django:DEFAULT_FROM_EMAIL`)
+DEFAULT_FROM_EMAIL = os.environ.get(
+    "INTEGREAT_COMPASS_SERVER_EMAIL", "keineantwort@integreat-compass.de"
+)
+
+#: The email address that error messages come from, such as those sent to :attr:`~integreat_compass.core.settings.ADMINS`.
+#: (see :setting:`django:SERVER_EMAIL`)
+SERVER_EMAIL = os.environ.get(
+    "INTEGREAT_COMPASS_SERVER_EMAIL", "keineantwort@integreat-compass.de"
+)
+
+#: A list of all the people who get code error notifications. When :attr:`~integreat_compass.core.settings.DEBUG` is ``False``,
+#: Django emails these people the details of exceptions raised in the request/response cycle.
+ADMINS = [("Integreat Compass Helpdesk", "tech@integreat-compass.de")]
+
+#: The host to use for sending email (see :setting:`django:EMAIL_HOST`)
+EMAIL_HOST = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST", "localhost")
+
+#: Password to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_HOST_PASSWORD`). If empty, Django won’t attempt authentication.
+EMAIL_HOST_PASSWORD = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST_PASSWORD")
+
+#: Username to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_HOST_USER`). If empty, Django won’t attempt authentication.
+EMAIL_HOST_USER = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST_USER", SERVER_EMAIL)
+
+#: Port to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_PORT`)
+EMAIL_PORT = int(os.environ.get("INTEGREAT_COMPASS_EMAIL_PORT", 587))
+
+#: Whether to use a TLS (secure) connection when talking to the SMTP server.
+#: This is used for explicit TLS connections, generally on port 587.
```

Comment:
This is copied from the Integreat settings.py, right? IMHO a better description would be `This will use Opportunistic TLS (STARTTLS command after starting a plain text connection).`